site2/docs/concepts-messaging.md

```diff
@@ -185,6 +185,21 @@ consumer.negativeAcknowledge(msg);
 > **Note**  
 > If batching is enabled, all messages in one batch are redelivered to the consumer.
 
+### Negative Redelivery Backoff
+
+In general, the consumer is not able to process the message successfully, what we can use [Negative acknowledgement](concepts-messaging.md#Negative acknowledgement) and redeliver the message after processing the message failure so that the message can be redelivered to other consumers(for the Shared subscription).
```

Comment:
```suggestion
In general, consumers are not able to process messages successfully. In this case, you can use [negative acknowledgement](concepts-messaging.md#negative-acknowledgement) and redeliver the messages after processing message failures, so that the messages can be redelivered to other consumers (for the Shared subscription).
```

do you mean this?